app/views/admin/applicants/index.html.haml

```diff
@@ -9,6 +9,7 @@
       %th= t('admin.applicants.index.table.descision')
       %th
   %tbody
+  <!-- I need to some how put a form here, wiht a submit button that gets and replaces @applicants -->

```

Comment:
This is in the **correct** view (not sure if it's in the correct place in that view, but who cares, this whole view is getting blown up anyways).

Reply:
Thanks, yeah I wasn't sure about comments in these kind of files heh

I'm planning to do something like this: https://guides.rubyonrails.org/form_helpers.html#a-generic-search-form for here, is that the right way to build a form for this?

Reply:
> I'm planning to do something like this: https://guides.rubyonrails.org/form_helpers.html#a-generic-search-form for here, is that the right way to build a form for this?

Seems legit to me 👍